rules_bazel/java/java_diff_test.bzl

```diff
@@ -39,7 +39,7 @@ junit_output_zip = rule(
         ),
     },
     outputs = {
-        "output": "%{name}%.zip",
+        "output": "%{name}.zip",
```

Comment:
`%{name}%.zip` will generate zip file named `test_output%.zip`, removed the extra `%`